x-pack/plugins/observability_solution/apm/public/utils/http_status_code_to_color.ts

```diff
@@ -5,35 +5,17 @@
  * 2.0.
  */
 
-import { euiLightVars as theme } from '@kbn/ui-theme';
-const { euiColorDarkShade, euiColorWarning } = theme;
+import { useEuiTheme } from '@elastic/eui';
 
-export const errorColor = '#c23c2b';
-export const neutralColor = euiColorDarkShade;
-export const successColor = '#327a42';
-export const warningColor = euiColorWarning;
-
-const httpStatusCodeColors: Record<string, string> = {
-  1: neutralColor,
-  2: successColor,
-  3: neutralColor,
-  4: warningColor,
-  5: errorColor,
+export const useGetStatusColor = (status: string | number) => {
+  const { euiTheme } = useEuiTheme();
+  const httpStatusCodeColors: Record<string, string> = {
```

Comment:
These are status/severity colors, we should use the vis colors here instead.

```
1: euiTheme.colors.vis.euiColorVisGrey0,
2: euiTheme.colors.vis.euiColorVisSuccess0,
3: euiTheme.colors.vis.euiColorVisGrey0,
4: euiTheme.colors.euiColorVisWarning0,
5: euiTheme.colors.euiColorVisDanger0,
```

Linking also the currently open proposal for severity colors as mentioned by @gvnmagni [here](https://github.com/elastic/kibana/pull/202985#issuecomment-2527546225)

Reply:
update made! Thanks!